py3status/modules/spaceapi.py

```diff
@@ -90,6 +96,15 @@ def check(self):
 
         return response
 
+    def on_click(self, event):
+        button = event['button']
+        if self.button_action and self.button_action == button:
```

Comment:
Here we could just use `if self.button_action == button:` as button can only be 1, 2, 3, 4 or 5.  I have used the longer version in the past so not a big deal.

It could also be combined with the `if self.url:`